modules/ROOT/pages/mtf-metadata-testing.adoc

```diff
@@ -0,0 +1,363 @@
+= Metadata 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Test the metadata defined for your module.
+
+== Before You Begin
+
+* Know how to define metadata in your module. For more information, refer to xref:metadata.adoc[].
+* To create metadata tests, add the `mtf-tools` dependency to your project:
```

Comment:
Do you need to specify the filename here, or is it obvious from the larger context?